### PR TITLE
Cleanup worker filter translations

### PR DESCRIPTION
**Pull request by lisbet-alvarez**
:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/https://hashicorp.atlassian.net/browse/ICU-15815)

# Description

Refactor worker filter related translations to reduce duplications of the same mappings.

## Screenshots (if appropriate)
Screenshot 2024-12-12 at 11 25 44 AM

Screenshot 2024-12-12 at 11 29 00 AM

Screenshot 2024-12-12 at 2 14 16 PM

## How to Test

1. Go to storage bucket form & validate translations are not broken
2. Go to credential store details & validate "Worker filter" tab translations are not broken
3. Go to target details & validate "Workers tab translations are not broken

## Checklist

- [ ] I have added before and after screenshots for UI changes
- [ ] I have added JSON response output for API changes
- [ ] I have added steps to reproduce and test for bug fixes in the description
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: addons/core/translations/actions/en-us.yaml [hunk 1]
```diff
@@ -63,3 +63,5 @@ narrow-results: Narrow results
 overflow-options: Overflow Options
 show-errors: Show Errors
 hide-errors: Hide Errors
+edit-worker-filter: Edit Worker Filter
+add-worker-filter: Add Worker Filter
```
Comment (lisbet-alvarez) [inline addons/core/translations/actions/en-us.yaml:67]: Each letter of new word is capitalized to match other actions.

File: addons/core/translations/form/en-us.yaml [hunk 2]
```diff
@@ -183,3 +183,5 @@ created:
 other_fields:
   label: Other Fields
   help: This information is provided by the server and cannot be edited in Boundary
+worker_filter:
+  label: Worker Filter
```
Comment (lisbet-alvarez) [inline addons/core/translations/form/en-us.yaml:187]: Each letter of word is capitalized, tho im noticing that in some form fields only the first letter is capitalized. This pattern should probably be standardized.

**Review by ZedLi — APPROVED** [review]
Thanks for the cleanup!

**Review by calcaide — APPROVED** [review]
Good work!! Thanks for the cleanup

**Review by cameronperera — APPROVED** [review]
nice changes!

**Pull request merged**